app/pages/HomePage/HomePage.tsx

```diff
@@ -11,14 +11,14 @@ import ResourceList from './components/ResourceList/ResourceList';
 
 const covidResources = [
   { name: 'Food', icon: 'food', categorySlug: 'food-resources' },
-  { name: 'Shelters', icon: 'bed', categorySlug: 'shelter-resources' },
-  { name: 'Showers, Hygiene and other Services', icon: 'shower', categorySlug: 'hygiene-resources' },
+  { name: 'Shelter', icon: 'bed', categorySlug: 'shelter-resources' },
+  { name: 'Showers and Restrooms', icon: 'shower', categorySlug: 'hygiene-resources' },
```

Comment:
And I think these changes accidentally revert https://github.com/ShelterTechSF/askdarcel-web/pull/1105.